api/src/main/java/io/opentelemetry/trace/BigendianEncoding.java

```diff
@@ -138,6 +138,15 @@ static void byteToBase16String(byte value, char[] dest, int destOffset) {
     byteToBase16(value, dest, destOffset);
   }
 
+  static byte[] bytesFromBase16(CharSequence value, int srcOffset, int charactersToRead) {
```

Comment:
```suggestion
  static byte[] bytesFromBase16(CharSequence value, int offset, int length) {
```

I think this pattern is very prevelant in the JDK so good to follow the idiom